dependencies.sh

```diff
@@ -8,7 +8,7 @@ export BYOND_MAJOR=515
 export BYOND_MINOR=1633
 
 #rust_g git tag
-export RUST_G_VERSION=3.7.0
+export RUST_G_VERSION=BRUH_FORGOR_TO_TICK_UP_THE_CLOCK
```

Comment:
```suggestion
export RUST_G_VERSION=3.8.0
```

Also rebuild rust_g.dll